Core/include/Acts/Vertexing/AdaptiveMultiVertexFinder.ipp

```diff
@@ -116,8 +116,10 @@ auto Acts::AdaptiveMultiVertexFinder<vfitter_t, sfinder_t>::find(
         break;
       }
     }
+    // MARK: fpeMaskBegin(FLTUND, 1, #2590)
     bool keepVertex = isGoodVertex &&
                       keepNewVertex(vtxCandidate, allVerticesPtr, fitterState);
```

Comment:
Could the mask go further in to `keepNewVertex`?

Reply:
the stacktrace did not point me further down and I didn't want to guess